config/custom_navajo.json

```diff
@@ -4,6 +4,18 @@
     "name": "5.1.0+trunk",
     "expiry": "2100-01-01"
   },
+  {
+    "url": "https://github.com/ocaml/ocaml/archive/4.14.tar.gz",
+    "name": "4.14+trunk+nnp",
+    "expiry": "2100-01-01",
+    "configure": "--disable-naked-pointers"
+  },
+  {
+    "url": "https://github.com/ocaml/ocaml/archive/4.14.tar.gz",
+    "name": "4.14+trunk+nnp+flambda",
+    "expiry": "2100-01-01",
```

Comment:
After we add support for 4.14, please update the PR to use a short-time interval, such as a week or ten days. Thanks!

Reply:
The idea is to follow the development branch for 4.14. But I suppose this depends on #6.

Reply:
Yes, that is correct!